src/intro-to-c/operators/operators-intro.md

```diff
@@ -0,0 +1,288 @@
+    printf("%d == %d is %d \n", a, c, a == c);
+    printf("%d != %d is %d \n", a, b, a != b);
+    printf("%d != %d is %d \n", a, c, a != c);
+    printf("%d > %d is %d \n", a, b, a > b);
+    printf("%d > %d is %d \n", a, c, a > c);
+    printf("%d < %d is %d \n", a, b, a < b);
+    printf("%d < %d is %d \n", a, c, a < c);
+    printf("%d >= %d is %d \n", a, b, a >= b);
+    printf("%d >= %d is %d \n", a, c, a >= c);
+    printf("%d <= %d is %d \n", a, b, a <= b);
+    printf("%d <= %d is %d \n", a, c, a <= c);
+
+    return 0;
+}
+```
+
+The output is:
+```
+3 == 3 is 1 
+3 == 7 is 0 
+3 != 3 is 0 
+3 != 7 is 1 
+3 > 3 is 0 
+3 > 7 is 0 
+3 < 3 is 0 
+3 < 7 is 1 
+3 >= 3 is 1 
+3 >= 7 is 0 
+3 <= 3 is 1 
+3 <= 7 is 1 
+```
+Remember: 1 is true, 0 is false.
+
+## Logical Operators
+A logical operator connects two or more expressions and returns either 0 or 1 depending upon whether the compound expression results true or false. 
+
+- **`&&`** : Logical AND, it is true only if all operands are true
+- **`||`** : Logical OR, it is true if either one operand is true
+- **`!`** : Logical NOT, it is used to reverse the logical state. If a condition is true, then NOT will make it false.
+
+Let's look at an example:
+```c
+#include <stdio.h>
+
+int main(void)
+{
+    int a = 3, b = 3, c = 7, result;
+
+    result = (a == b) && (c > b);
+    printf("(a == b) && (c > b) is %d \n", result);
+
+    result = (a == b) && (c < b);
+    printf("(a == b) && (c < b) is %d \n", result);
+
+    result = (a == b) || (c < b);
+    printf("(a == b) || (c < b) is %d \n", result);
+
+    result = (a != b) || (c < b);
+    printf("(a != b) || (c < b) is %d \n", result);
+
+    result = !(a != b);
+    printf("!(a != b) is %d \n", result);
+
+    result = !(a == b);
+    printf("!(a == b) is %d \n", result);
+
+    return 0;
+}
+```
+The output is
+```
+(a == b) && (c > b) is 1 // both expressions are true, so the compound expression is true
+(a == b) && (c < b) is 0 // the first expression is true but the second is not, so the compound expression is false
+(a == b) || (c < b) is 1 // the first expression is true, so the compound expression is true
+(a != b) || (c < b) is 0 // both expressions are false, so the compound expression is false
+!(a != b) is 1 // a != b will return 0 but since we have the !, then !0 is 1
+!(a == b) is 0 // a == b will return 1 but since we have the !, then !1 = 0
+```
+
+
+## Bitwise Operators
+Bitwise operators perform bit-level operations. This is not used in intro to C, but it will be a good reference for the future.
+- **`&`** : Bitwise AND, copies a bit to the result if it exists in both operands
+- **`|`** : Bitwise OR, copies a bit if it exists in either operand.
+- **`^`** : Bitwise XOR, copies the bit if it is set in one operand but not both.
+
+So to understand this example, I recommend you have a binary converter opened or if you have taken CDA with Almaki, you will be pretty familiar with binary convertions. 
```

Comment:
I would remove/edit this line. It refers to a professor that not everyone has taken, especially if these materials become pushed to an audience outside of UCF, though the mild jab at that class is pretty good. If you want to keep the reference to using a binary converter, adding a link to one would be really nice. I found a good one here: https://bitwisecmd.com/#help